src/operator/nn/fully_connected.cc

```diff
@@ -87,8 +90,16 @@ void FullyConnectedComputeExCPU(const nnvm::NodeAttrs& attrs,
     return;
   }
   FallBackCompute(FullyConnectedCompute<cpu>, attrs, ctx, inputs, req, outputs);
+#else
+  std::vector<TBlob> in_blobs(inputs.size());
+  for (size_t i = 0; i < in_blobs.size(); i++) in_blobs[i] = inputs[i].data();
+  std::vector<TBlob> out_blobs(outputs.size());
+  for (size_t i = 0; i < out_blobs.size(); i++) out_blobs[i] = outputs[i].data();
+  FullyConnectedCompute<cpu>(attrs, ctx, in_blobs, req, out_blobs);
+#endif
```

Comment:
I think ‘FallBackCompute’ should be used to fall back the computation to original cpu implementation.

Reply:
This block will only be executed when MKL is absent

Reply:
When USE_MKL=1,
I wanted to simply use the non-MKL FCForward with 
```
data.data(),
weight.data(),
bias.data(),
```
assuming `data.data()` returns a TBlob with normal cpu layout even if data is in MKL layout. `weight.data()` and `bias.data()` should always return normal cpu layout if weight and bias are `row_sparse`. Please advice.